networking_arista/ml2/arista_sec_gp.py

```diff
@@ -325,6 +329,56 @@ def _get_interface_membership(self, server, ports):
 
         return membership
 
+    def _get_portchannel_membership(self, server, portchannel):
+        """Get portchannel membership information for a list of ports"""
+        if portchannel not in self._PORTCHANNEL_MEMBERSHIP[server]:
+            self._refresh_interface_membership(server)
+
+        return self._PORTCHANNEL_MEMBERSHIP[server][portchannel]
+
+    def _get_mlag_neighbor_server(self, server):
+        """Get the server() for an mlag neighbor of a server()"""
+        # find mlag neighbor
+        mlag_detail = server(["show mlag detail"])
+        if not (mlag_detail and mlag_detail[0]):
+            LOG.warning("Could not fetch mlag details on %s", server)
+            return None
+        neigh = EUI(mlag_detail[0]['detail']['peerMacAddress'])
+        neigh_server = self._get_server(self, switch_id=neigh)
+        if not neigh_server:
+            LOG.warning("Could not find mlag neighbor %s in switch list",
+                        neigh, server)
```

Comment:
Doesn't that log line need another `%s` somewhere?

Reply:
Yes, nice find! Fixed it